src/sdk/actions.ts

```diff
@@ -96,12 +108,12 @@ export async function createActionsPlugin<TConfig = unknown, TEnv = unknown, TSu
     }
 
     if (pluginOptions.postCommentOnError && loggerError) {
-      await postComment(context, loggerError);
+      await postErrorComment(context, loggerError);
     }
   }
 }
 
-async function postComment(context: Context, error: LogReturn) {
+async function postErrorComment(context: Context, error: LogReturn) {
```

Comment:
there's another `postComment` function in `server.ts`, maybe we can merge these 2?